packages/purpose-process/src/model/domain/errors.ts

```diff
@@ -23,13 +24,17 @@ export const errorCodes = {
   purposeVersionCannotBeDeleted: "0009",
   organizationIsNotTheProducer: "0010",
   notValidVersionState: "0011",
-  eServiceModeNotAllowed: "0012",
-  missingFreeOfChargeReason: "0013",
-  riskAnalysisValidationFailed: "0014",
-  purposeNotInDraftState: "0015",
-  duplicatedPurposeTitle: "0016",
+  missingRejectionReason: "0012",
```

Comment:
This error has been removed in a previous PR

Reply:
Updated, thanks